### ISSUE TITLE
run_sketch utility won't leave the window open if the sketch has setup() but no draw()

### ISSUE THREAD

**Issue by hx2A** [py5coding/py5generator#89]
When I put this in Thonny and run it:

``` python
def setup():
    size(400, 400)
    rect(10, 10, 20, 20)
```

The window appears for a split second and then exits. If I add this:

``` python
def draw():
    pass
```

Then the window appears. Need the same dummy draw method trick I use for static sketches.

**Comment by tabreturn** [comment]
Seems like an odd scenario -- like, why bother with `setup()` at all? Then again, Processing.py will resort to static mode behavior -- so your dummy draw proposal replicates that :+1:

**Comment by villares** [comment]
> What if they don't know about functions yet?

Yep, that's a challenge. Some noise they are putting up with!
"Trust me, put your code indented into this setup() block here and py5 will call this for you. I'll explain this def thing later."
It might indeed be a bad idea, I'm not sure. I had to use it with pyp5js and rationalized it probably.

**Comment by hx2A** [comment]
> Yep, that's a challenge. Some noise they are putting up with!

Maybe consider using py5bot to help with that! Here's a notebook I used to do just that in a intro presentation two weeks ago:

https://github.com/py5coding/py5examples/blob/main/Python%20with%20py5bot/Intro%20to%20Python%20with%20py5bot.ipynb

**Comment by hx2A** [comment]
When I put this in Thonny and run it:

```python
def setup():
    size(400, 400, P2D)
    rect(10, 10, 20, 20)
```

the window stays open, as it should. This bug only affects Sketches that use the default renderer.

Also, this bug is related to #82. A proper fix for #82 will fix this bug as well. Any fix for this issue that does not also fix #82 is a hack.

**Comment by hx2A** [comment]
I fixed two seemingly unrelated bugs by removing one line of code. :)
